app/src/main/java/org/oppia/app/walkthrough/WalkthroughPages.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.walkthrough
+
+/** Represents current state of Walkthrough Activity. */
```

Comment:
Please update comment to `/** Represents current state of [WalkthroughActivity]. */ `